gramps/gen/lib/test/grampstype_test.py

```diff
@@ -42,7 +43,7 @@ class GT0(GrampsType):
 # A migration utility might instantiate several of these with
 #   varying blacklist-specs
 class GT1(GT0):
-    _BLACKLIST = BLIST
+    _BLACKLIST: list[int] | None = BLIST
```

Comment:
Instead of typing on `list` (and `dict`, both in general and not just on this particular line), can we instead type on `collections.Sequence`, `collections.MutableSequence`, `collections.Mapping`, and `collections.MutableMapping`? In my experience it makes things slightly less dependent on internals (one historical example was `dict` being ordered as an internal implementation detail until... 3.7?). Concrete benefits are also that return values can be marked immutable, reducing the risk of other code attempting to modify data that should be read-only.